test_demo.py

```diff
@@ -8,10 +8,16 @@ def test_show_aggie_pride(self):
         df = show_aggie_pride()
         self.assertEqual(df.loc[0, 'Text'], 'Aggie Pride')
         self.assertEqual(df.loc[1, 'Text'], 'Worldwide')
+
```

Comment:
It doesn't look like this will work because you have two slogans in the same integer location (loc=2)